src/notifications/notificationmanager.cpp

```diff
@@ -1208,6 +1224,22 @@ void NotificationManager::fetchData(bool update)
         hints[id].insert(hintName, value);
     }
 
+    // Gather the internal hints
+    QSqlQuery internalHintsQuery("SELECT * FROM internal_hints", *m_database);
+    QSqlRecord internalHintsRecord = internalHintsQuery.record();
+    int internalHintsTableIdFieldIndex = hintsRecord.indexOf("id");
```

Comment:
This has `hintsRecord` instead of `internalHintsRecord`. That sounds wrong to me even if it happens to work. Is it a copy-paste mistake? (See also lines below.)

Reply:
Duh, yes. Worked as the field order is the same.

While at it, I added a second commit for removing "Field" part from all these names. Some are already halfway unreadable even without it.